components/Form/index.js

```diff
@@ -66,9 +72,24 @@ export default function Form({ onSubmit }) {
       formObject.image = image;
     }
 
-    localStorage.setItem("favoriteLocation", JSON.stringify(formObject));
+    formObject.id = Date.now().toString();
```

Comment:
This makes for a better id value.
```suggestion
    formObject.id = Math.random().toString(32).substring(2);
```